.circleci/config.yml

```diff
@@ -0,0 +1,187 @@
+# (c) Copyright 2019 Hewlett Packard Enterprise Development LP
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2.1
+
+workflows:
+  build and publish:
+    jobs:
+      - build-container
+      - verify-container:
+          requires:
+            - build-container
+      - publish-container-hub:
+          requires:
+            - verify-container
+      - publish-container-quay:
+          requires:
+            - verify-container
+
+defaults:
+  - restore_cache: &python_deps_restore_cache
+      keys:
+        - &python_deps_cache_key v1-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "test-requirements.txt" }}
+        - v1-dependencies-
+
+  - save_cache: &python_deps_save_cache
+      paths:
+        - "~/.tox"
+        - "~/.cache/pip"
+      key: *python_deps_cache_key
+
+jobs:
+
+  build-container:
+    docker:
+      - image: circleci/python:3.6
+
+    working_directory: ~/repo
```

Comment:
Use default working directory?

Reply:
Will update and see if that works, thanks